package/contents/ui/main.qml

```diff
@@ -73,31 +78,32 @@ Item {
 	}
 
 	// https://stackoverflow.com/questions/4842424/list-of-ansi-color-escape-sequences
-	property var ansiColors: ({
-		30: '#000000', // Black
-		31: '#aa0000', // Red
-		32: '#00aa00', // Green
-		33: '#aa6500', // Yellow
-		34: '#0000aa', // Blue
-		35: '#aa00aa', // Magenta
-		36: '#00aaaa', // Cyan
-		37: '#aaaaaa', // White
-		90: '#656565', // Bright Black
-		91: '#ff6565', // Bright Red
-		92: '#65ff65', // Bright Green
-		93: '#ffff65', // Bright Yellow
-		94: '#6565ff', // Bright Blue
-		95: '#ff65ff', // Bright Magenta
-		96: '#65ffff', // Bright Cyan
-		97: '#ffffff', // Bright White
-	})
 	function resetState(state) {
 		var out = state.closeTags.join(' ')
 		state.bold = false
 		state.closeTags = []
 		return out
 	}
 	function parseAnsiCode(n, i, tokens, state) {
+		var ansiColors = ({
+			30: '#000000', // Black
+			31: '#aa0000', // Red
+			32: '#00aa00', // Green
+			33: '#aa6500', // Yellow
+			34: '#0000aa', // Blue
+			35: '#aa00aa', // Magenta
+			36: '#00aaaa', // Cyan
+			37: '#aaaaaa', // White
+			90: '#656565', // Bright Black
+			91: '#ff6565', // Bright Red
+			92: '#65ff65', // Bright Green
+			93: '#ffff65', // Bright Yellow
+			94: '#6565ff', // Bright Blue
+			95: '#ff65ff', // Bright Magenta
+			96: '#65ffff', // Bright Cyan
+			97: '#ffffff', // Bright White
+		})
+
```

Comment:
I don't understand why this was duplicated and moved into the functions itself instead of remaining a property? In the function the HashMap is created every function call while when it's outside we only do a lookup.

Reply:
Also if it's outside we can eventually make it themeable which we can't do if it's hardcoded in the function.